File: rafsun_jani/1295/findEven.java
class Solution {
    public static int findNumbers(int[] nums) {
        int count = 0;
        for(int i =0; i<nums.length; i++){
            long n = nums[i];
            int digitCount = 0;
            while(n != 0){
                n /=10;
                digitCount++;
            }
            if(digitCount % 2 == 0){
                count++;
            }

        }
        return count;
    }

    public static void main(String[] args){
        int[] numbers = {10, 300, 23};
        System.out.print(findNumbers(numbers));
    }
}

File: saif_akib/aug25/1295/findEven.js
//TC: O(n)
//MC: O(1)
var findNumbers = function(nums) {
    let count = 0;
    nums.forEach((num) => {
        let length = num.toString().split("").length;
        if(length % 2 == 0) count++; 
    })
    return count;
};

const arr = [12, 345, 2, 6, 7896];
console.log(findNumbers(arr));